src/vendor-no-composer/robinthehood/ModifiedStdModule/Classes/StdModule.php

```diff
@@ -386,11 +386,26 @@ protected function getConfig(string $name, $default = false): string
      */
     protected function getVersion(): string
     {
-        $tempVersion = $this->tempVersion ?? '';
-        if (!$tempVersion) {
-            $this->tempVersion = $this->getConfig('VERSION');
+        $version = $this->tempVersion ?? '';
+
+        if ('' !== $version) {
+            return $version;
         }
-        return $this->tempVersion;
+
+        $versionConstant = \sprintf('%s_VERSION', $this->modulePrefix);
+        $versionQuery = \xtc_db_query(
+            \sprintf(
+                'SELECT `configuration_value`
+                   FROM `%s`
+                  WHERE `configuration_key` = "%s"',
+                \TABLE_CONFIGURATION,
+                $versionConstant
+            )
+        );
+        $versionData = \xtc_db_fetch_array($versionQuery);
+        $version = $versionData['configuration_value'] ?? '';
```

Comment:
We are in the global Namespace so we do not need the `\` - prefix for `sprintf` and `xtc_...` or even `\TABLE_CONFIGURATION`

Reply:
Sorry, I didn't express myself well. The methods, functions and constants are defined in the global namespace and therefore do not need a \ as a prefix when calling. However, we can also leave it as it is. I have just done some research. Apparently the PHP interpreter is supposed to find the right method more quickly this way and it should be clearer for the developer which namespace the call is coming from. However, it is still not entirely clear to me whether a \ is used for every standard PHP function. For me, this makes the code look less attractive. It doesn't seem to me that it has been done this way in other projects either, or I have just never paid attention to it because OOP programming is so good that the standard functions are always outsourced to small class methods.

Reply:
>Apparently the PHP interpreter is supposed to find the right method more quickly this way

Oh wow, TIL!

>However, it is still not entirely clear to me whether a \ is used for every standard PHP function.

I have made a habit of using it. There are some exceptions as far as I know (`isset`, I think).

> For me, this makes the code look less attractive. It doesn't seem to me that it has been done this way in other projects either

Yes, I agree. But due to the explicit nature of it, I think it's worth it. Especially if it helps the PHP interpreter to run faster.